pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dto/queue/SkippedParticipantEventDto.java

```diff
@@ -0,0 +1,22 @@
+package org.broadinstitute.dsm.db.dto.queue;
```

Comment:
I pulled these out of the old `ParticipantEvent` to separate object from db transactions